website/docs/r/virtual_hub_ip_configuration.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_virtual_hub_ip_configuration"
+description: |-
+  Manages a Virtual Hub IP Configuration.
+---
+
+# azurerm_virtual_hub_ip_configuration
+
+Manages a Virtual Hub IP Configuration.
```

Comment:
I'm not sure what a `Virtual Hub IP Configuration` is. Is this an IP in the virtual hub, is this configuring something specific?

Reply:
I assume it's an IP in the virtual hub.